### ISSUE TITLE
Trouble caused by installing linux610-headers and v4l2loopback-dkms

### ISSUE THREAD

**Issue by HenriLaurie** [JuliaLang/IJulia.jl#1133]
I wonder if anyone has seen this before: IJulia notebook closed immediately after opening, without even launching a browser tab.

The cause was, as best I can tell, the following sequence of actions

1. I installed linux610-headers and v4l2loopback-dkms from the shell, via `sudo pacman -S linux610-headers` and `sudo pacman -S v4l2loopback-dkms`
2. I ran OBStudio and added camera to captured devices.
3. At the Julia prompt, I issued `using IJulia` and `notebook()` and found that it failed to launch a browser tab and simply showed the message when IJulia returns.

I cured it by first uninstalling the two packages I had added. Still failed. By looking at the jupyter version I saw that notebook hadn't been installed, so I used Manjaro's Add/Remove Software utility to add it. This also added nbconvert, which had also been missing. However, I think I now have a hybrid version of jupyter. It seems to work for what I have tried but some of the menus look decidedly different.

Is this an IJulia issue, or a Manjaro (perhaps ArchLinux?) issue? Any comments appreciated!

**Comment by janxkoci** [comment]
Have you installed Julia from Manjaro repos? I did and it worked with jupyterlab directly, without even installing IJulia. But some packages failed to compile, and then I learned the version in repos is [unofficial](https://wiki.archlinux.org/title/Julia) and known to fail. I'm currently looking into getting it working without resorting to AUR, but running out of options... (My attempt at https://github.com/JuliaLang/IJulia.jl/issues/1139)

**Comment by HenriLaurie** [comment]
No, I use juliaup,  which I installed per the instructions on  https://github.com/JuliaLang/juliaup. It seems that IJulia just defaults to a jupyter notebook, I haven't tested this in detail but I'm making some notebooks for use in lectures and I am not sure that the markdown is IJulia markdown and not just Jupyter markdown.

Just back from holidays and slowly re-educating myself about this issue ...

**Comment by janxkoci** [comment]
I got both `notebook()` and `jupyterlab()` available from IJulia. I will probably try the officially recommended way next, at the moment I tried `juliaup` from `conda` manager (I use it for everything at work) and I have no kernel in notebooks...

**Comment by HenriLaurie** [comment]
The recommendation is  NOT to use anything except the Windows store (on Windows machines) or directly from julialang.org on Unix-like machines. They give the code to use on the command line (no command  line if you go directly to the Windows store, for  which they give a link).

Not sure where you can get full details on why this the current recommendation from the juliaup people.

**Comment by janxkoci** [comment]
I'm aware of all of that, but thanks

**Comment by janxkoci** [comment]
In the end rebuilding IJulia fixed everything for me, now I'm happily using the version from julia-forge installed with mamba 😊
